src/main/scala/no/ndla/learningpathapi/model/search/SearchableLearningPath.scala

```diff
@@ -20,5 +20,6 @@ case class SearchableLearningPath(id: Long,
                                   lastUpdated: Date,
                                   tags: SearchableTags,
                                   author: String,
-                                  learningsteps: Seq[SearchableLearningStep])
+                                  learningsteps: Seq[SearchableLearningStep],
+                                  isBasedOn: Option[Long])
```

Comment:
`SearchableLearningPath` brukes for å indeksere læringsstien i elasticsearch. Jeg er litt usikker på om `isBasedOn` også burde være med i elastic mappingen (`SearchIndexServiceComponent.scala` `buildMapping()`). Vil tro @kstigen har fasiten